js/components/dataset/resource/form.vue

```diff
@@ -101,6 +101,7 @@ export default {
             type: Object,
             default() {return new Resource()}
         },
+        notifications: true
```

Comment:
This is the only one with positive variable (ie. `notifications` vs `hideNotifications`).
We should stay consistent and chose between one of the naming (personnaly I prefer `notifications` over `hideNotifications`)